app/src/main/java/io/zenandroid/onlinego/data/db/Database.kt

```diff
@@ -6,17 +6,34 @@ import androidx.room.RoomDatabase
 import androidx.room.TypeConverters
 import io.zenandroid.onlinego.data.model.local.*
 import io.zenandroid.onlinego.data.model.ogs.JosekiPosition
+import io.zenandroid.onlinego.data.model.ogs.PuzzleRating
+import io.zenandroid.onlinego.data.model.ogs.PuzzleSolution
 
 /**
  * Created by alex on 04/06/2018.
  */
 @Database(
-        entities = [Game::class, Message::class, Challenge::class, GameNotification::class, JosekiPosition::class, HistoricGamesMetadata::class, ChatMetadata::class, ChallengeNotification::class],
-        version = 18,
+        entities = [
+            Game::class,
+            Message::class,
+            Challenge::class,
+            GameNotification::class,
+            JosekiPosition::class,
+            HistoricGamesMetadata::class,
+            ChatMetadata::class,
+            ChallengeNotification::class,
+            PuzzleCollection::class,
+            Puzzle::class,
+            PuzzleRating::class,
```

Comment:
why do we need to put the rating in a different table?

Reply:
It's a separate API call and object, it just seemed simpler. Are you thinking it's worth merging it into Puzzle?

Reply:
We do offer that actually, they can be rated from the Tsumego screen, but I'll see about merging them

Reply:
Just a thought - I think the reason I kept those separate was because if the user logs out and logs in as another account, the puzzle table can be reused, but the ratings will all need to be refreshed because they're per-user. Is that enough of a reason to keep them as they are?